docs/man/encrypt/_index.md

```diff
@@ -34,6 +34,9 @@ command:
     - name: kas-url-path
       description: URL path to the KAS service at the platform endpoint domain. Leading slash is required if needed.
       default: /kas
+    - name: target-mode
+      description: The target tdf spec version
+      default: ""
```

Comment:
Please add an example here. Also, what does the validation look like? Does the SDK provide an error message we can key off on to improve CLI UX in the event of a bad version being passed and failing?
```suggestion
    - name: target-mode
      description: The target tdf spec version (i.e. "4.1")
```

Reply:
updated the docs, it uses the Masterminds/semver package to handle the version comparison (this handles the semver validation), if invalid semver is provided you get an error like `Failed to encrypt: NewTDFConfig failed: isLessThanSemver failed: semver.NewVersion failed for version 4.nope.3: Invalid Semantic Version`

Reply:
should we add a note that this may be removed in the future?

Reply:
good idea, added